scripts/bulk-mailer.js

```diff
@@ -6,291 +6,48 @@
 
 'use strict'
 
-var commandLineOptions = require('commander')
-var config = require('../config').getProperties()
-var fs = require('fs')
-var leftpad = require('leftpad')
-var log = require('../lib/log')(config.log.level, 'bulk-mailer')
-var Mailer = require('../mailer/index')
-var nodeMailerMock = require('./bulk-mailer/nodemailer-mock')
-var P = require('../lib/promise')
-var path = require('path')
+const bulkMailer = require('./bulk-mailer/index')
+const program = require('commander')
+const path = require('path')
 
-commandLineOptions
+program
   .option('-b, --batchsize [size]', 'Number of emails to send in a batch. Defaults to 10', parseInt)
   .option('-d, --delay [seconds]', 'Delay in seconds between batches. Defaults to 5', parseInt)
-  .option('-e, --errors [filename]', 'JSON output file that contains errored emails. Defaults to ./errors.json')
-  .option('-i, --input <filename>', 'JSON input file')
-  .option('-t, --template <template>', 'Template filename to render')
-  .option('-u, --unsent [filename]', 'JSON output file that contains emails that were not sent. Defaults to ./unsent.json')
+  .option('-i, --input <filename>', 'JSON user input file')
+  .option('-m, --method <method>', 'Sender method to call')
```

Comment:
Something to consider, certain sender methods can send multiple template types. I don't think there is anything actionable but just wanted to call that out.